src/main/java/twilightforest/TwilightForestMod.java

```diff
@@ -88,15 +88,8 @@ public void postInit(FMLPostInitializationEvent evt) {
 			DimensionManager.registerDimension(TFConfig.dimension.dimensionID, TwilightForestMod.dimType);
 		} else {
 			TwilightForestMod.LOGGER.warn("Detected that the configured dimension id '{}' is being used.  Using backup ID.  It is recommended that you configure this mod to use a unique dimension ID.", TFConfig.dimension.dimensionID);
-			DimensionManager.registerDimension(TwilightForestMod.backupdimensionID, TwilightForestMod.dimType);
-			TFConfig.dimension.dimensionID = TwilightForestMod.backupdimensionID;
-		}
-
-		if (Loader.isModLoaded("Thaumcraft")) {
-			//FIXME: Reenable this once Thaumcraft is available.
-			//registerThaumcraftIntegration();
```

Comment:
If you're going to remove this remove the actual `registerThaumcraftIntegration` method as well